test/_init_tests.js

```diff
@@ -160,6 +166,10 @@ class TestConfig {
     return this.if(this.platform.isIos.bind(this.platform));
   }
 
+  ifIe() {
+    return this.if(this.platform.isIe.bind(this.platform));
```

Comment:
I thought the line I added to the constructor to skip IE by default (line 104) would do what we wanted. I'll re-read the code to see why that isn't the case.